content/english/python-earsketch/1-meet-dj-nuevo/_index.md

```diff
@@ -5,12 +5,15 @@ date: 2019-07-23T11:45:38-07:00
 weight: 1
 draft: false
 ---
-<iframe width="100%" height="600px" src="https://www.youtube.com/embed/g0u1CkbpUWQ?start=79" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 # Nuevo has new headphones! Can you find yours? 
 You got contacted to make Nuvi's next hit mixtape, but when you get to the studio, Nuvi says their next project is going to be produced all in code. This is something that hasn't been done before, are you up to the challenge? Look around yourself to find your headphones and get ready to
 join him!
 
 Before we start, make sure that your headphones are working. 
 
-[comment]: <> (Try playing the following audio. What do you hear?! If you are reading this, it is because your browser does not support the audio element : audio/good-enough.mp3)
\ No newline at end of file
+[comment]: <> (Try playing the following audio. What do you hear?! If you are reading this, it is because your browser does not support the audio element : audio/good-enough.mp3)
+
+### Video Walkthrough
```

Comment:
@Anvesha4 did we want to have the videos at the top or the bottom of the page? I don't mind either (although I do like it at the top, maybe we can put it in another panel?)